datalab/datalab_session/file_utils.py

```diff
@@ -81,15 +81,15 @@ def create_jpgs(cache_key, fits_paths: str, color=False) -> list:
 
 def stack_arrays(array_list: list):
   """
-  Takes a list of numpy arrays, crops them to an equal shape, and stacks them to be a 3d numpy array
-
+  Takes a list of numpy arrays from fits images and stacks them to be a 3d numpy array
+  cropped since fits images can be different sizes
   """
-  min_shape = min(arr.shape for arr in array_list)
-  cropped_data_list = [arr[:min_shape[0], :min_shape[1]] for arr in array_list]
+  min_x = min(arr.shape[0] for arr in array_list)
+  min_y = min(arr.shape[1] for arr in array_list)
 
-  stacked = np.stack(cropped_data_list, axis=2)
+  cropped_data_list = [arr[:min_x, :min_y] for arr in array_list]
 
-  return stacked
+  return np.stack(cropped_data_list, axis=2)
```

Comment:
This comment doesn't have to do with testing/this PR but is more general about stacking. Cropping to the smallest dimension probably doesn't make sense, because if the images are different sizes but of the same target the cropped portions wouldn't be the portions that would overlap, but then again you would need to do registration, not just stacking, to actually align them. Right now we only use this method in Median and RGB stacking operations, and both of those should probably require the images to come from the same instrument and therefor have the same size, or else the result wouldn't really make sense anyway.

Reply:
agreed, current cropping assumes that the images are aligned at 0,0 which in most cases wouldn't make sense. Should keep this in mind when writing a registering function